beginner_source/nlp/advanced_tutorial.py

```diff
@@ -217,7 +217,7 @@ def _forward_alg(self, feats):
     def _get_lstm_features(self, sentence):
         self.hidden = self.init_hidden()
         embeds = self.word_embeds(sentence).view(len(sentence), 1, -1)
-        lstm_out, self.hidden = self.lstm(embeds)
+        lstm_out, self.hidden = self.lstm(embeds, self.hidden)
```

Comment:
@cmusjtuliuyuan init_hidden is still required to reset the model. init_hidden can now just `return None`.

But I guess we'll leave it as it is to make things clear and explicit.